pkg/core/workflow/types.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package workflow includes all type definitaions about Workflow.
+package workflow
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Workflow defines the root structure of a workflow.
+type Workflow struct {
+	gorm.Model
+	UID         string         `gorm:"index:uid" json:"uid"`
+	Name        string         `json:"name"`
```

Comment:
We might not use `namespace + name` instead of UID as a unique identity, just mentioning, it's not so important. 😁

Reply:
Know about it. This conversation can be saved here to remind others who view this PR in the future.

Reply:
Know about it 👌🏽.